### PR TITLE
Convert debug manager includes

### PR DESCRIPTION
**Pull request by Arcasias**
Enterprise: https://github.com/odoo-dev/enterprise/pull/152

### CODE REVIEW

File: addons/point_of_sale/static/src/js/debug_manager.js [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+                target: "new",
+                type: "ir.actions.act_url",
+                url: "/pos/ui/tests?mod=*",
+            });
+        },
+        sequence: 35,
+    };
+}
+
+registry.category("debug").category("global").add("point_of_saile.runPoSJSTests", runPoSJSTests);
```
Comment (ged-odoo) [inline addons/point_of_sale/static/src/js/debug_manager.js]: lol at "saile"

Comment (Arcasias) [reply addons/point_of_sale/static/src/js/debug_manager.js]: Woops~

File: addons/mail/static/src/js/tools/debug_manager.js [hunk 2]
```diff
@@ -1,30 +1,38 @@
...
-//             domain: [['res_id', '=', selectedIDs[0]], ['model', '=', this._controller.modelName]],
-//             context: {
-//                 default_res_model: this._controller.modelName,
-//                 default_res_id: selectedIDs[0],
-//             },
-//         });
-//     },
-// });
+export function manageMessages({ action, component, env }) {
+    console.log({ action, component, env });
```
Comment (ged-odoo) [inline addons/mail/static/src/js/tools/debug_manager.js]: console.log

Comment (aab-odoo) [inline addons/mail/static/src/js/tools/debug_manager.js]: console.log :-)

File: addons/web/static/src/core/debug/debug_menu.js [hunk 3]
```diff
@@ -10,51 +10,62 @@ const { Component } = owl;
 
 export class DebugManager extends Component {
```
Comment (ged-odoo) [inline addons/web/static/src/core/debug/debug_menu.js]: DebugMenu

File: addons/web/static/src/core/debug/debug_menu.js [hunk 4]
```diff
@@ -71,17 +82,21 @@ export class DebugManager extends Component {
 DebugManager.debugElementsId = 1;
 DebugManager.template = "web.DebugManager";
 
-export function useDebugManager(elementsFactory) {
-    const elementsId = DebugManager.debugElementsId++;
+export function useDebugManager(categories, context = {}) {
```
Comment (ged-odoo) [inline addons/web/static/src/core/debug/debug_menu.js]: useDebugMenu

**Comment by aab-odoo** [comment]
robodoo rebase-ff r+

**Comment by aab-odoo** [comment]
robodoo retry

**Pull request closed without merging**